### PR TITLE
up versioned orchestrator-core to 4.13.1

### LINKED ISSUE #1212: Update orchestrator-core version to 4.13.1 to include fix for iculib issue in MacOS

### PR DESCRIPTION
**Pull request by tsuwandy**
Fixes #1212 #1217

### CODE REVIEW

**Review by joshgummersall — APPROVED** [review]

**Review by daveta — APPROVED** [review]

File: packages/orchestratorlib/src/build.ts [hunk 1]
```diff
@@ -282,11 +282,14 @@ export class OrchestratorBuild {
   private static async processLuConfig(luConfig: any, labelResolvers: Map<string, LabelResolver>, fullEmbeddings: boolean = false): Promise<any[]> {
     const luObjects: any[] = [];
     for (const file of (luConfig.models || [])) {
-      const luObject: any = {
-        content: OrchestratorHelper.readFile(file),
-        id: path.basename(file, '.lu'),
-      };
-      luObjects.push(luObject);
+      const content: string = OrchestratorHelper.readFile(file);
+      if (content && content.length > 0) {
```
Comment (munozemilio) [inline packages/orchestratorlib/src/build.ts:286]: if (content)  should work too

File: packages/orchestratorlib/src/orchestratorhelper.ts [hunk 2]
```diff
@@ -1123,16 +1122,16 @@ export class OrchestratorHelper {
     (idsToFind || []).forEach((ask: any)  => {
       const resourceToFind: string = path.isAbsolute(ask.filePath) ? ask.filePath : path.resolve(path.join(baseDir, ask.filePath));
       const fileContent: string = OrchestratorHelper.readFile(resourceToFind);
-      if (!processedFiles.includes(resourceToFind)) {
-        processedFiles.push(resourceToFind);
-      }
-      if (fileContent) {
+      if (fileContent && fileContent.length > 0) {
```
Comment (munozemilio) [inline packages/orchestratorlib/src/orchestratorhelper.ts:1125]: if (fileContent) should work

File: packages/orchestratorlib/src/orchestratorhelper.ts [hunk 3]
```diff
@@ -1150,10 +1149,13 @@ export class OrchestratorHelper {
     } else {
       const ext: string = path.extname(inputPath);
       if (ext === '.lu') {
-        retPayload.push({
-          content: OrchestratorHelper.readFile(inputPath),
-          id: path.basename(inputPath, '.lu'),
-        });
+        const content: string = OrchestratorHelper.readFile(inputPath);
+        if (content && content.length > 0) {
```
Comment (munozemilio) [inline packages/orchestratorlib/src/orchestratorhelper.ts:1153]: Same as before

**Review by munozemilio — APPROVED** [review]
Minor comments

File: packages/orchestratorlib/src/orchestratorhelper.ts (hunk 3 above)
Comment (tsuwandy) [reply packages/orchestratorlib/src/orchestratorhelper.ts]: fixed

File: packages/orchestratorlib/src/orchestratorhelper.ts (hunk 2 above)
Comment (tsuwandy) [reply packages/orchestratorlib/src/orchestratorhelper.ts]: fixed

**Pull request merged**